src/components/GoogleAddressLookup/__test__/format.spec.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
this tests should be placed inside helpers folder, so inside helpers you create another __test__ folder and then there is where the helpers test should be placed